client/web/src/components/exit-node-selector.tsx

```diff
@@ -252,18 +270,25 @@ function ExitNodeSelectorItem({
   return (
     <button
       key={node.ID}
-      className="w-full px-4 py-2 flex justify-between items-center cursor-pointer hover:bg-stone-100"
+      className={cx(
+        "w-full px-4 py-2 flex justify-between items-center hover:bg-stone-100",
+        {
+          "text-gray-400 cursor-not-allowed": !node.Online,
+        }
+      )}
       onClick={onSelect}
+      disabled={!node.Online}
```

Comment:
Still could technically let them select it... Do we disable for other platforms?

Reply:
We disable it on both macOS and iOS

Reply:
The important one is that if a node had selected an exit node which subsequently goes offline, that we not automatically disable the exit node and start sending traffic in the clear. Doing so would open up an attack surface area to try to force traffic into the clear.

Reply:
this only controls whether the UI element is selected.  We don't change the behavior of the exit node at all here.  I've tested and confirmed that an active exit node that goes offline just results in a traffic black hole.